### PR TITLE
Sync config with branch

### PR DESCRIPTION
**Pull request by manute**
https://github.com/InVisionApp/elroy/issues/262

### CODE REVIEW

File: src/lib/elroy-sync.js [hunk 1]
```diff
@@ -86,7 +86,9 @@ class ElroySync {
         // TODO: include resource data in config
         if (!resource.disable) {
           cluster.resources[name] = {
-            config: {}
+            config: {
```
Comment (c1freitas) [inline src/lib/elroy-sync.js]: I would just send every value under the resource since we will eventually need it all.

Comment (manute) [reply src/lib/elroy-sync.js]: ok good point. Shall I check for `undefineds` here 😨  ?

File: src/lib/elroy-sync.js [hunk 2]
```diff
@@ -86,7 +86,10 @@ class ElroySync {
...
+              branch: resource.branch,
+              containers: resource.containers
```
Comment (manute) [inline src/lib/elroy-sync.js]: I have seen also the property `file` and `kind` for ingress , but I guess we don't need them

Comment (c1freitas) [reply src/lib/elroy-sync.js]: I think we need it all. Might be best to loop over all the resource objects properties adding them all. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyNames

**Pull request merged**